Docs/LinuxOsxInstall.md

```diff
@@ -31,3 +20,31 @@ In general, ensure you have SDL2, OpenGL and dotnet 6 or later. Use the `Yafc` e
 
 ### Flathub
 Note that [the version available on Flathub](https://flathub.org/apps/details/com.github.petebuffon.yafc) is not the Community Edition. Its repo can be found at https://github.com/petebuffon/yafc. 
+
+## OSX installation instructions
+Below are the instructions for the arm64 Macs. To use them for Intel Macs, please apply the changes to the folder `Yafc/lib/osx` instead of `Yafc/lib/osx-arm64`.
+- [Install dotnet core (v8.0 or later)](https://dotnet.microsoft.com/download)
+- Download the lua5.2.1 source, copy the .patch file and apply.
```

Comment:
Link to `lua5.2.1` source? Also, what/where is the `.patch`?

Reply:
Updated the instructions with the link.